modules/bbmap/index/main.nf

```diff
@@ -19,10 +19,10 @@ process BBMAP_INDEX {
     }
 
     input:
-    tuple val(meta), path(fasta)
+    path fasta
 
     output:
-    tuple val(meta), path('ref')  , emit: index
+    path 'ref'                    , emit: index
     path "*.version.txt"          , emit: version
```

Comment:
```suggestion
    path 'ref'          , emit: index
    path "*.version.txt", emit: version
```

Reply:
I didn't see this before I merged I'm afraid @drpatelh.

Reply:
Don't worry. I was going to commit it myself but didn't want to mess with your `git` foo :)